Makefile

```diff
@@ -0,0 +1,128 @@
+
+ifeq ("$(wildcard project)","")
+$(error project directory doesn't exist)
+endif
+
+ifneq ("$(wildcard project/app.mk)","")
+include project/app.mk
+endif
+
+# --------------------------------------------------------------------------------------------------
+# Variables
+# --------------------------------------------------------------------------------------------------
+
+ifndef APP_NAME
+$(error APP_NAME is not set in the app.mk file)
+endif
+
+ifndef APP_DESCRIPTION
+$(error APP_DESCRIPTION is not set in the app.mk file)
+endif
+
+GIT_SUMMARY := $(shell cd project && git describe --tags --dirty --always)
+GIT_BRANCH := $(shell cd project && git rev-parse --abbrev-ref HEAD)
+BUILD_STAMP := $(shell date -u '+%Y-%m-%dT%H:%M:%S%z')
+
+DOCKER_REGISTRY ?= registry.uw.systems
+DOCKER_NAMESPACE ?= partner
+MAIN_IMAGE_NAME := $(subst $(DOCKER_NAMESPACE)-,,$(APP_NAME))
+DOCKER_BASE_NAME ?= $(DOCKER_REGISTRY)/$(DOCKER_NAMESPACE)
+
+ifeq ($(GIT_BRANCH), master)
+    DOCKER_TAG := latest
+else
+    DOCKER_TAG := $(GIT_BRANCH)
+endif
+
+LDFLAGS := -ldflags '-s \
+	-X "github.com/utilitywarehouse/partner-pkg/meta.ApplicationName=$(APP_NAME)" \
```

Comment:
Repo is for generic go circle build yet we're pulling in partner specific items

Reply:
I can either move partner specific things to the `ap.mk` file in the repo or we can rename this repo to be partner specific.

Reply:
Lets just make it partner specific (rename) we can make generic if others buy into it